src/main/java/seedu/address/logic/commands/ingredientcommands/IngredientViewSingleCommand.java

```diff
@@ -0,0 +1,113 @@
+package seedu.address.logic.commands.ingredientcommands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INGREDIENT;
+
+import java.util.Optional;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.SetCommand;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ingredient.Ingredient;
+import seedu.address.model.ingredient.IngredientName;
+
+/**
+ * View the level of a single specific ingredient.
+ */
+public class IngredientViewSingleCommand extends Command {
+
+    public static final String COMMAND_WORD = "i-view-single";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " :view a single ingredient level in tCheck."
```

Comment:
Perhaps it should be `views` because a command is singular.

Reply:
For example, i-set is i-set, so I guess this is more consistent..?

Reply:
noted with thanks

Reply:
Oh, I am referring to the description. It should be "views a single ingredient ... ", since the command word is singular